docs/REFERENCE-environment_variables.md

```diff
@@ -97,6 +97,7 @@
 | ROLLBAR_CLIENT_TOKEN                             | Client token for Rollbar error tracking.                                                                                                                                                                                                                                                                                                                                                    |
 | ROLLBAR_ACCESS_TOKEN                             | Access token for Rollbar error tracking.                                                                                                                                                                                                                                                                                                                                                    |
 | ROLLBAR_ENDPOINT                                 | Endpoint URL for Rollbar error tracking.                                                                                                                                                                                                                                                                                                                                                    |
+| SEND_AUTO_OPT_OUT_RESPONSE                           | Send the organization's default opt-out message if a user is auto-opted out. Alternatively, set a shouldAutoRespond property in an item for the auto opt-out regex to true, for it to only apply on those regex matches.                                                                |
```

Comment:
`Alternatively, set a shouldAutoRespond property in an item for the auto opt-out regex to true, for it to only apply on those regex matches.`

Won't the setting the SEND_AUTO_OPT_OUT_RESPONSE also only apply on the regex matches in the auto opt-out regex?

Reply:
Yes, I just discovered that you can have multiple regex for the customized auto-optouts :)